### ISSUE TITLE
auto_rendering locks view in k3d 2.9.1

### ISSUE THREAD

**Issue by mironvanderkolk** [K3D-tools/K3D-jupyter#255]
In 2.8.2 I could set `auto_rendering` keyword argument of `k3d.plot`  to `False` and freely rotate the view with the mouse. I use this to multiple changes and then explicitly call `k3d.plot.render()` for optimization purposes

In version 2.9.2 setting `auto_rendering` to `False` leads to the view being stuck (am able to correctly add objects though)

Code to reproduce:
```
import k3d
k3d.plot(auto_rendering=False)
```
Jupyterlab info:
```
JupyterLab v2.2.9
Known labextensions:
   app dir: /home/miron/.pyenv/versions/3.8.7/envs/deps-update-jan/share/jupyter/lab
        @jupyter-voila/jupyterlab-preview v1.1.0  enabled  OK
        @jupyter-widgets/jupyterlab-manager v2.0.0  enabled  OK
        ipysheet v0.4.4  enabled  OK
        jupyter-matplotlib v0.7.4  enabled  OK
        jupyterlab-datawidgets v7.0.0  enabled  OK
        jupyterlab-plotly v4.14.1  enabled  OK
        k3d v2.9.1  enabled  OK
        plotlywidget v4.14.1  enabled  OK
```

Relevant python package version:
```
ipydatawidgets==4.2.0
ipykernel==5.4.3
ipython==7.19.0
ipython-genutils==0.2.0
ipywidgets==7.6.2
jupyter-client==6.1.11
jupyter-core==4.7.0
jupyter-server==1.2.1
jupyterlab==2.2.9
jupyterlab-pygments==0.1.2
jupyterlab-server==1.2.0
jupyterlab-widgets==1.0.0
K3D==2.9.1
nbclient==0.5.1
nbconvert==6.0.7
nbformat==5.0.8
notebook==6.1.6
numpy==1.19.5
traitlets==5.0.5
traittypes==0.2.1
widgetsnbextension==3.5.1
```

**Comment by artur-trzesiok** [comment]
Will be fixed in 2.9.2

**Comment by artur-trzesiok** [comment]
https://pypi.org/project/K3D/
